src/components/ModuleSidebar/ModuleSidebar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Sidebar from '../Sidebar/Sidebar';
+import sideLogo from '../../assets/images/logo.png';
+import './moduleSidebar.scss';
+
+const ModuleSidebar = () => (
+  <Sidebar>
+    <div className="d-aside">
+      <div>
+        <span className="d-logo">
+          <img src={sideLogo} alt="logo" />
+        Community
+        </span>
+      </div>
+      <ul className="d-links">
+        <li className="module-active"><Link to="/dashboard">Modules</Link></li>
+        <li><Link to="/profile">Profile</Link></li>
+        <li><Link to="/payment">Payment</Link></li>
+        <li><Link to="/aboutus">Help</Link></li>
+      </ul>
+    </div>
+  </Sidebar>
```

Comment:
Curious about why we have to wrap all this within the Sidebar component?. WEIRD!!!

Reply:
Okay, so we were thinking since there are going to be different Sidebars appearing in different pages of the app, (e.g the sidebar in the Course Dashboard would be different from the Sidebar in the Community page), so we decided to create a re-usable Sidebar component that would just take in child elements as props.  

So the same sidebar would render different items on different pages.

Reply:
But you're only nesting elements here, you're not passing any props. I think this sidebar is not very reusable. How can we refactor and improve this?